Inline Assist keybinding configuration on the setting panel

catrielmuller opened the pull request:
## Context

Resolve #1410 
Resolve #1417 

This PR adds configurable keybinding controls for the Inline Assist feature, allowing users to enable or disable keyboard shortcuts for quick inline tasks and auto inline tasks through the settings panel. The implementation extends the Ghost service settings with new boolean configuration options and properly integrates them into the VS Code extension context system.

## Implementation

- Adds UI controls for toggling Cmd+I (Quick Inline Task) and Cmd+L (Auto Inline Task) keybindings
- Updates the extension's package.json to conditionally enable keybindings based on user settings
- Implements proper state management and context updates for the new configuration options

## Screenshots

<img width="638" height="762" alt="Screenshot 2025-07-22 at 4 58 23 PM" src="https://github.com/user-attachments/assets/e02f09ff-b2fe-416c-9eca-84e740f804ce" />


## How to Test

Enable or dissable the Inline Assitant keybinding on the Setttings panel

catrielmuller committed:
feat: inline assist keybinding configurations

hassoncs reviewed (commented):
left a few #nit-y comments around messaging and hardcoded keyboard keys, but I think this is exactly what we need!

catrielmuller committed:
refactor: add translations

catrielmuller committed:
Update .changeset/curvy-times-camp.md

Co-authored-by: Chris Hasson <hassoncs@gmail.com>

Commit:
feat(ui): add keybinding configuration to GhostService settings

Introduces the ability to open global keybindings from the GhostService settings. This allows users to easily configure keyboard shortcuts for inline assist features.

- Adds `openGlobalKeybindings` message type to `WebviewMessage`.
- Implements `openGlobalKeybindings` handler in `webviewMessageHandler` to execute VS Code command.
- Updates `GhostServiceSettings.tsx` to include links to open keybindings, with optional filters for specific commands.
- Modifies i18n files across all locales to include new strings for keybinding descriptions and triggers.

hassoncs commented:
Just pushed the keyboard shortcuts link:
![2025-07-22 09 37 47](https://github.com/user-attachments/assets/99db5264-d8ce-4a75-a35c-0d4a2955c1dc)

Commit:
Merge remote-tracking branch 'origin/main' into catrielmuller/disable-inline-assist

Commit:
remove unused string key
